demos/bazel/README.md

```diff
@@ -0,0 +1,46 @@
+# Bazel Reactive gRPC
+
+This demo shows how to use the Reactive-gRPC Bazel rules.
+
+## Setup
+
+Include this project as an external dependency in your `WORKSPACE`.
```

Comment:
Looks great! Can you please re-include the comment about `protocopt=--include_source_info`, so users will know why their comments are missing?

`Add `build --protocopt=--include_source_info` to your `.bazelrc` file.`

Reply:
Done, I changed it slightly so users will know that it is not a required step